### PR TITLE
Update juicer_postprocessing.sh

### PR DESCRIPTION
**Pull request by adadiehl**
Expose --threads option for Arrowhead

### CODE REVIEW

File: CPU/common/juicer_postprocessing.sh [hunk 1]
```diff
@@ -84,7 +85,7 @@ fi
 
 echo -e "${juicer_tools_path} is post-processing Hi-C for ${genomeID}\nData read from ${hic_file_path}.\nMotifs read from ${bed_file_dir}\n"
 echo -e "ARROWHEAD:\n"
-${juicer_tools_path} arrowhead ${hic_file_path} ${hic_file_path%.*}"_contact_domains"
+${juicer_tools_path} arrowhead ${hic_file_path} ${hic_file_path%.*}"_contact_domains" --threads ${threads}
```
Comment (sa501428) [inline CPU/common/juicer_postprocessing.sh]: can you move `--threads ${threads}` to come right after `arrowhead` in the line?

**Review by sa501428 — CHANGES_REQUESTED** [review]
Can you move the threads flag to come right after the arrowhead command?

File: CPU/common/juicer_postprocessing.sh (hunk 1 above)
Comment (adadiehl) [reply CPU/common/juicer_postprocessing.sh]: Done

**Review by sa501428 — APPROVED** [review]

**Comment by pna059** [comment]
With or without the -z/--threads option postprocessing is not working:
```
$ bash ~/Juicer_dir/scripts/common/juicer_postprocessing.sh -j ~/Juicer_dir/scripts/common/juicer_tools -i aligned/inter.hic -g ~/Juicer_dir/chrom_sizes/hv03.chr.sizes
/home/pavlan/Juicer_dir/scripts/common/juicer_tools is post-processing Hi-C for /home/pavlan/Juicer_dir/chrom_sizes/hv03.chr.sizes
Data read from aligned/inter.hic.
Motifs read from /gpfs0/juicer2/references/motif

ARROWHEAD:

Exception in thread "main" jargs.gnu.CmdLineParser$IllegalOptionValueException: Illegal value 'aligned/inter.hic' for option -z/--threads
        at jargs.gnu.CmdLineParser$Option$IntegerOption.parseValue(CmdLineParser.java:202)
        at jargs.gnu.CmdLineParser$Option.getValue(CmdLineParser.java:157)
        at jargs.gnu.CmdLineParser.parse(CmdLineParser.java:485)
        at jargs.gnu.CmdLineParser.parse(CmdLineParser.java:426)
        at juicebox.tools.HiCTools.main(HiCTools.java:61)
***! Problem while running Arrowhead
```
With the option:
```
$ bash ~/Juicer_dir/scripts/common/juicer_postprocessing.sh -j ~/Juicer_dir/scripts/common/juicer_tools -z 10 -i aligned/inter.hic -g ~/Juicer_dir/chrom_sizes/hv03.chr.sizes/
/home/pavlan/Juicer_dir/scripts/common/juicer_postprocessing.sh: illegal option -- z
Usage: /home/pavlan/Juicer_dir/scripts/common/juicer_postprocessing.sh [-h] -j <juicer_tools_file_path> -i <hic_file_path> -m <bed_file_dir> -g <genome ID>
```

**Comment by adadiehl** [comment]
Sorry, realize now I did not provide -t with a default value. Will get a
patch up ASAP.

**Pull request merged**